src/main/webapp/topic/editTopic.xhtml

```diff
@@ -0,0 +1,31 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:h="http://java.sun.com/jsf/html"
+	xmlns:f="http://java.sun.com/jsf/core"
+	xmlns:ui="http://java.sun.com/jsf/facelets">
+<h:head>
+	<h:outputStylesheet library="css" name="style.css" />
+	<title>Modifica</title>
+</h:head>
+<h:body>
+	<h:form>
+		<h3>Modifica Topic</h3>
+		<h:panelGrid columns="2" border="0">
+			Id:
+			<h:inputText id="modificaId"
```

Comment:
Se vuoi puoi mettere tutto il tag inputText in una riga sola e tieni sempre questo standard ovunque.

Reply:
fatto